Formula/testssl.rb

```diff
@@ -3,14 +3,14 @@ class Testssl < Formula
   homepage "https://testssl.sh/"
   url "https://github.com/drwetter/testssl.sh/archive/v3.0.8.tar.gz"
   sha256 "22c5dc6dfc7500db94b6f8a48775f72b5149d0a372b8552ed7666016ee79edf0"
-  license "GPL-2.0-or-later"
+  license "GPL-2.0-only"
```

Comment:
Didn't see any "or any later..." language in source. Main script is just:
```
# License: GPLv2, see https://www.fsf.org/licensing/licenses/info/GPLv2.html
# and accompanying license "LICENSE.txt". Redistribution + modification under this
# license permitted.
```

I don't think there is a compatibility issue here as it is a script that runs `openssl` rather than a binary that links to `openssl`.